engines/suppliers/Gemfile.lock

```diff
@@ -2,68 +2,93 @@ PATH
   remote: .
   specs:
     suppliers (0.0.1)
-      rails (~> 4.1.7)
+      rails (~> 4.2.5)
```

Comment:
Yeah but for now please stick with what we're doing with others engines and we'll fix it all in a once. :pray: